app/views/foreman/job_templates/flatpak_setup.erb

```diff
@@ -0,0 +1,37 @@
+<%#
+kind: job_template
+name: Install Flatpak and Firefox
+job_category: Katello
+description_format: 'Install Flatpak and Firefox'
+feature: katello_host_flatpak_install
```

Comment:
If you're introducing a new REX feature, you'll have to register the feature (in plugin.rb I think?)

You don't have to use a feature, if you don't need easy access from Hammer or web UI..